cg/apps/crunchy.py

```diff
@@ -0,0 +1,320 @@
+        if not self.get_index_path(cram_path).exists():
+            LOG.info("No index-file for %s", cram_path)
+            return False
+        if not flag_path.exists():
+            LOG.info("No %s file for %s", FLAG_PATH_SUFFIX, cram_path)
+            return False
+        return True
+
+    def bam_compression_possible(self, bam_path: Path) -> bool:
+        """Check if it CRAM compression for BAM file is possible"""
+        if bam_path is None or not bam_path.exists():
+            LOG.warning("Could not find bam %s", bam_path)
+            return False
+        if self.cram_compression_done(bam_path=bam_path):
+            LOG.info("cram compression already exists for %s", bam_path)
+            return False
+        return True
+
+    #
+    # def spring_compression_done(
+    #     self, fastq_first_path: Path, fastq_second_path: Path
+    # ) -> bool:
+    #     """Check if spring compression already is done for fastq-files"""
+    #
+    #     spring_path = self.change_suffix_spring(
+    #         fastq_first_path=fastq_first_path, fastq_second_path=fastq_second_path
+    #     )
+    #     flag_path = spring_path.with_suffix(FLAG_PATH_SUFFIX)
+    #
+    #     if not spring_path.exists():
+    #         LOG.info("No spring file %s", spring_path)
+    #         return False
+    #     if not flag_path.exists():
+    #         LOG.info("No %s file for %s", FLAG_PATH_SUFFIX, spring_path)
+    #         return False
+    #     return True
+
+    # def fastq_compression_possible(
+    #     self, fastq_first_path: Path, fastq_second_path: Path
+    # ) -> bool:
+    #     """Check if spring compression is possible for fastq-files"""
+    #     if fastq_first_path is None or not fastq_first_path.exists():
+    #         LOG.warning("Could not find fastq %s", fastq_first_path)
+    #         return False
+    #     if fastq_second_path is None or not fastq_second_path.exists():
+    #         LOG.warning("Could not find fastq %s", fastq_second_path)
+    #         return False
+    #     if self.spring_compression_done(
+    #         fastq_first_path=fastq_first_path, fastq_second_path=fastq_second_path
+    #     ):
+    #         return False
+    #
+    #     return True
+
+    @staticmethod
+    def get_flag_path(file_path):
+        return file_path.with_suffix(FLAG_PATH_SUFFIX)
+
+    @staticmethod
+    def get_index_path(file_path):
+        index_type = CRAM_INDEX_SUFFIX
+        if file_path.suffix == BAM_SUFFIX:
+            index_type = BAM_INDEX_SUFFIX
+        return file_path.with_suffix(file_path.suffix + index_type)
+
+    @staticmethod
+    def get_cram_path_from_bam(bam_path: Path) -> Path:
+        """ Get corresponding CRAM file path from bam file path """
+        if not bam_path.suffix == BAM_SUFFIX:
+            LOG.error("%s does not end with %s", bam_path, BAM_SUFFIX)
+            raise ValueError
+        cram_path = bam_path.with_suffix(CRAM_SUFFIX)
+        return cram_path
+
+    # @staticmethod
```

Comment:
branch and delete here?

Reply:
I think not done things should not go into production, I still thinks we should remove this dangling outcommented code, what do you say?